src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/DomainEventService.kt

```diff
@@ -251,6 +265,7 @@ class DomainEventService(
     BookingCancelledEnvelope::class.java -> DomainEventType.APPROVED_PREMISES_BOOKING_CANCELLED
     BookingChangedEnvelope::class.java -> DomainEventType.APPROVED_PREMISES_BOOKING_CHANGED
     ApplicationWithdrawnEnvelope::class.java -> DomainEventType.APPROVED_PREMISES_APPLICATION_WITHDRAWN
+    AssessmentAppealedEnvelope::class.java -> DomainEventType.APPROVED_PREMISES_APPLICATION_APPEAL_MADE
```

Comment:
did you mean to use "assessment" rather than "application" in the domain event type:

`AssessmentAppealedEnvelope::class.java -> DomainEventType.APPROVED_PREMISES_ASSESSMENT_APPEAL_MADE` 

I think that would be more accurate and have less potential to cause confusion...